File: 240721/배열의 특정 위치 찾기/find-specific-location-fo-array.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int evenSum = 0;

        int mutiplesOfThreeSum = 0;
        int cnt = 0;

        for(int i = 0; i < 10; i++){
            int num = sc.nextInt();

            if(i % 2 != 0){
                evenSum += num;
            }

            if((i + 1) % 3 == 0){
                cnt++;
                mutiplesOfThreeSum += num;
            }
        }

        System.out.printf("%d %.1f", evenSum, mutiplesOfThreeSum / (double)cnt);

        
    }
}

File: 240721/배열의 특정 위치 찾기/find-specific-location-fo-array.js
const fs = require('fs');
const arr = fs.readFileSync(0).toString().trim().split(' ').map(Number);

let evenSum = 0;

let mutiplesOfThreeSum = 0;
let cnt = 0;

arr.forEach((number, idx) => {
    if(idx % 2 !== 0){
        evenSum += number;
    }

    if((idx+1) % 3 === 0){
        cnt++;
        mutiplesOfThreeSum += number
    }
})

console.log(evenSum, (mutiplesOfThreeSum / cnt).toFixed(1));